app/controllers/users.rb

```diff
@@ -0,0 +1,28 @@
+class ChitterApp < Sinatra::Base
+
+  get '/users/new' do
+    erb :'users/new'
+  end
+
+  post '/users' do
+    user = User.create(email:                 params[:email],
+                       name:                  params[:name],
+                       username:              params[:username],
+                       password:              params[:password],
+                       password_confirmation: params[:password_confirmation])
+    if user.save
+      session[:id] = user.id
+      flash.next[:message] = "Succesfully registered #{user.username}, you may now login below:"
```

Comment:
Metrics/LineLength: Line is too long. [96/90] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)